repo/fsrepo.go

```diff
@@ -431,12 +489,15 @@ func isEmptyDir(path string) (bool, error) {
 	return len(infos) == 0, nil
 }
 
-func fileExists(file string) bool {
+func fileExists(file string) (bool, error) {
```

Comment:
Is this functionality not anywhere else in the code base?  (for that matter why isn't this entire thing part of golang, but that's another issue)